views/site.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+   <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header><h1>Welcome to the fan site of Lewis Hamilton</h1> 
+        <div class="head"></div>
+    </header>
+    <link rel="js" href="app.js">
+    <h2>Lewis is one of the greatest F1 racers of all time, and for many the G.O.A.T </h2>
+    <article class="button">
+        <span>
+        <form action="/about" method="get" target="_blank">
```

Comment:
Here it would've been better if you used `<a>`. You also forgot to close each form, so the first one works, and the rest doesn't.